### DISCUSSION TITLE
tesserract is not working captcha

### DISCUSSION THREAD

**Discussion by indraraj26** [naptha/tesseract.js#958] [Q&A] [answered]
Hi Team,

I don't know why it is not working with this captcha. 

![image](https://github.com/user-attachments/assets/367155d9-1922-4517-87f4-3ac65da7e152)

I tried it on https://tesseract.projectnaptha.com/ there also it returns nothing

Thank you

**Comment by Balearica** [comment, marked as answer]
It looks like the issue is that this is white text on an transparent background, which presumably Tesseract is not equipped to handle.  My guess is that Tesseract interprets transparent as equivalent to white, so it just ends up being a white image.

If you preprocess the image such that this is black text on a white background, Tesseract should be able to recognize this.  This would need to happen in your application given that, as a rule, we don't develop new features for Tesseract.js around the use-case of captchas.
